app/sidekiq/event_bus_gateway/letter_ready_email_job.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module EventBusGateway
+  class LetterReadyEmailJob
+    include Sidekiq::Job
+    include SentryLogging
+
+    # Job should not retry
+    sidekiq_options retry: 0
+    NOTIFY_SETTINGS = Settings.vanotify.services.event_bus_gateway
+    MAILER_TEMPLATE_ID = NOTIFY_SETTINGS.template_id.decision_letter_ready_email
+
+    # For now, receive arguments and send the email.
+    # Going forward, this will be a recurring job that pulls
+    # ready-to-send decision letter emails from the DB
+
+    # EVENTUAL BEHAVIOR
+    # return unless should_perform?
+
+    # send_ready_decision_letter_emails
+
+    # nil
+    def perform(participant_id, template_id, personalisation)
+      notify_client.send_email(
+        recipient_identifier: { id_value: participant_id, id_type: 'ICN' },
```

Comment:
I think we'll need to use `id_type: 'PID'`?  I'm not 100%, not sure of the options/format on that parameter 👀

Reply:
acknowledging that we did not figure this out yet, but we have eyes on it